Ruta get product by id

ImanolMalamud opened the pull request.

ImanolMalamud committed:
Ruta get product by id

danielva77 reviewed (commented):
Fijate, si la ruta puede quedar en un único archivo que sea tipo "ProductId.js". Para que no se pise con lo de Tuti y lo mio

ImanolMalamud commented:
Ok, ahora lo cambio. Despues se lo puede juntar en un archivo _**productRoutes.js**_ que tenga todas las rutas del producto.

ImanolMalamud committed:
Rutas:
- El index.js queda como estaba antes.
- archivo productRoutes.js creado
- get productos/:id agregado en productRoutes.js
